test/scripts/install-contour-working.sh

```diff
@@ -44,17 +44,10 @@ if ! kind::cluster::exists "$CLUSTERNAME" ; then
     exit 2
 fi
 
-# Wrap sed to deal with GNU and BSD sed flags.
-run::sed() {
-    local -r vers="$(sed --version < /dev/null 2>&1 | grep -q GNU && echo gnu || echo bsd)"
-    case "$vers" in
-        gnu) sed -i "$@" ;;
-        *) sed -i '' "$@" ;;
-    esac
-}
+# Set the image tag to match the current git hash + dirty flag.
+VERSION=$(git describe --exclude="*" --always --dirty)
```

Comment:
I noticed an issue with this change, repeat runs of `make install-contour-working` if you've made local changes no longer trigger a Deployment rollout because nothing has changed in the spec, whereas previously the tag was changing. We could explicitly run a `kubectl rollout restart`, or set an annotation on the Deployment, or explicitly delete the pods, or ...

Reply:
No worries, just realized it today when troubleshooting why my updated code wasn't running!

Yeah, maybe it's best to go back to random for now to keep that repeated deployment use case working, and we can revisit a long-term solution that addresses all the issues.

Reply:
Fixed for now in #5278